ui/app/mainui/AppMain.qml

```diff
@@ -654,13 +657,10 @@ Item {
 
                     Connections {
                         target: Global
-                        onOpenCreateChatView: {
-                            createChatView.opened = true
-                        }
-                        onCloseCreateChatView: {
-                            createChatView.opened = false
-                        }
+                        onOpenCreateChatView: createChatView.opened = true
```

Comment:
This syntax is deprecated and generates tons of warnings; pls use `function onFoo(params)` syntax instead

Reply:
Fixed